README.md

```diff
@@ -0,0 +1,50 @@
+# cqm-execution
+
+This repository contains an npm module for calculating eCQMs (electronic clinical quality measures) written in CQL (clinical quality language), using the Project Tacoma CQM models (https://github.com/projecttacoma/cqm-models).
+
+## Installation
+
+### npm
+```
+npm install cqm-execution
+```
+
+### yarn
+```
+yarn add cqm-execution
+```
+
+## Usage
+Example usage:
+```
+const Mongoose = require('mongoose');
```

Comment:
As part of this review, it would be great if someone can test this on their own machine as well!
@mayerm94